resources/migrations/001-flow.down.sql

```diff
@@ -0,0 +1,9 @@
+drop table consumer_offset;
```

Comment:
👍 

on the uppercase sql.. the rest of this project uses lowercase so I'd rather change all the casing in a separate PR if that's what we want.